hubdle-gradle-plugin/test/resources/config/documentation/changelog/1/CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
 
 ### Updated
 
+- `com.gradle.enterprise:com.gradle.enterprise.gradle.plugin -> com.gradle:com.gradle.develocity-gradle-plugin`
+
+### Updated
```

Comment:
As this changelog belongs to a test, I don't think it is necessary to update it (maybe the test will fail).

The main changelog is in the root directory. The one for the dependencies is in `hubdle-version-catalog` directory. I think we should mention this change in both changelogs.

Reply:
Yep, the test fails https://github.com/JavierSegoviaCordoba/hubdle/actions/runs/8633806051/job/23669028569?pr=910